lib/src/form_builder_validators.dart

```diff
@@ -172,6 +172,48 @@ class FormBuilderValidators {
     };
   }
 
+  /// [FormFieldValidator] that requires the words count of the field's value to be
+  /// greater than or equal to the provided minimum count.
+  static FormFieldValidator<String> minWordsCount(
+    int minCount, {
+    bool allowEmpty = false,
+    String? errorText,
+  }) {
+    assert(minCount > 0);
+    return (valueCandidate) {
+      int valueWordsCount = 0;
+
+      if (valueCandidate != null) {
+        if (valueCandidate.trim().isEmpty) {
+          valueWordsCount = 0;
+        } else {
+          valueWordsCount = valueCandidate.trim().split(' ').length;
+        }
+      }
+
+      return valueWordsCount < minCount && (!allowEmpty || valueWordsCount > 0)
```

Comment:
This solution can be more more like `maxWordsCount`. Like:
```dart
int valueWordsCount = valueCandidate?.trim().split(' ').length ?? 0;
return valueWordsCount < minCount && (!allowEmpty || valueWordsCount > 0)
```

Reply:
This one is more concise but when the `valueCandidate` is empty the length would be `1` rather than `0`.

Reply:
Ok, so maybe can simplify the first if, like this:
```dart
if (valueCandidate != null && valueCandidate.trim().isNotEmpty) {
   valueWordsCount = valueCandidate.trim().split(' ').length;
}
return valueWordsCount < minCount && (!allowEmpty || valueWordsCount > 0)
```

Reply:
Done ✅